platform/darwin/src/MGLAttributedExpression.h

```diff
@@ -5,7 +5,7 @@ NS_ASSUME_NONNULL_BEGIN
 typedef NSString * MGLAttributedExpressionKey NS_EXTENSIBLE_STRING_ENUM;
 
 FOUNDATION_EXTERN MGL_EXPORT MGLAttributedExpressionKey const MGLFontNamesAttribute;
-FOUNDATION_EXTERN MGL_EXPORT MGLAttributedExpressionKey const MGLFontSizeAttribute;
+FOUNDATION_EXTERN MGL_EXPORT MGLAttributedExpressionKey const MGLFontScaleAttribute;
 FOUNDATION_EXTERN MGL_EXPORT MGLAttributedExpressionKey const MGLFontColorAttribute;
```

Comment:
While we’re at it, these three constants and the typedef above should all be documented so that they show up in the jazzy docset. The documentation should state the type of object that the expression should evaluate as. Also, for `MGLFontScaleAttribute`, we should emphasize that the scale is relative to the overall attributed string’s font size as specified in `MGLSymbolStyleLayer.textFontSize`.